label.py

```diff
@@ -58,17 +57,50 @@ def close(e):
     root.quit()
 
 
-get_new_image()
-root.bind("y", callback_y)
-root.bind("d", callback_d)
-root.bind("n", callback_n)
-root.bind("<Return>", callback_skip)
-root.bind("<Escape>", close)
+def start_labeling():
+    global images_to_show, panel1, panel2, frame
+
+    frame.destroy()
+    labels = utils.read_labels(labels_directory + label_file_name.get())
+    images_to_show = [i for i in utils.get_images() if i not in labels]
+
+    panel1 = Label(root)
+    panel1.pack(side="left", padx=10)
+    panel2 = Label(root)
+    panel2.pack(side="left", padx=10)
+
+    get_new_image()
+    root.bind("y", callback_y)
+    root.bind("d", callback_d)
+    root.bind("n", callback_n)
+    root.bind("<Return>", callback_skip)
+    root.bind("<Escape>", close)
+
+
+def choose_file():
+    global frame, label_file_name
+
+    if not os.path.exists(labels_directory):
+        os.makedirs(labels_directory)
+    all_file_names = [f for f in os.listdir(labels_directory)]
+
+    frame = Frame(root)
+    label_file_name = StringVar(frame)
+    frame.pack(side="top")
+    input_type = Label(frame, text="Choose or add filename with .csv extension")
+    input_type.pack(side="top")
+    if len(all_file_names) != 0:
+        old_user = OptionMenu(frame, label_file_name, *all_file_names)
```

Comment:
The option menu is a little confusing, we should add some kind of placeholder.

Or, we can improve the form by doing something like:
Choose existing label file: [OPTIONMENU]
Create new label file: [TEXTINPUT]

Or, we can use argparse and pass the name of the label file as a parameter to the label.py script (so you would run it , for example, as `python label.py marco`) and remove this `choose_file` function.

All three options are fine by me, so feel free to decide which one you prefer.